sklearn/preprocessing/data.py

```diff
@@ -480,6 +480,14 @@ def minmax_scale(X, feature_range=(0, 1), axis=0, copy=True):
 class StandardScaler(BaseEstimator, TransformerMixin):
     """Standardize features by removing the mean and scaling to unit variance
 
+    The standard score of a data point `x` is calculated as:
+
+        z = (x - u) / std
+
+    where `u` is the mean of the population or zero if `with_mean=False`, and
```

Comment:
The term `population` is rather vague. What about `mean of the training samples`?

Reply:
Sounds better, thanks @TomDLT